### DISCUSSION TITLE
Error: "storage size of myHandles isn't known"

### DISCUSSION THREAD

**Discussion by Laiaborrell** [TheAlbertDev/MASB#51] [Q&A] [answered]
When debugging my code of last session (7) this error keeps appearing. I have already asked some classmates and no one has a clue of what could be happening. Also, I have reviewed that I have not left anything from the code, but obviously I am missing something.

![image](https://user-images.githubusercontent.com/78802600/114599434-60050780-9c93-11eb-9c24-3cbc30cc218b.png)

Any help please?
Thankyou!

**Comment by Albert-Alvarez** [comment]
What was the problem/solution?

**Comment by Laiaborrell** [comment]
I just did all the session again, but could not find any differences between the code which gave the error and the new one. Now it does not show any errors when debugging. However, it is still not working. Nothing happens when I send the " 0201010101010103D03F010101010103E03F010101010114E0BF027B14AE47E17A843F7B14AE47E17A743F00" code from the Cool Term in "Hex" although I have made sure that the baudrate is set at 115200 and that the breakpoint is placed correctly in the NOP() function. Also, the uart is enabled. I have no clue of what could be happening.
Laia

**Comment by Albert-Alvarez** [comment, marked as answer]
I have regenerated the code from the `.ioc` file and at least the microcontroller has entered in the `HAL_UART_RxCpltCallback` when a message from CoolTerm is sent.

Anyway, the program doesn't work because all the index increments are missing: `rxBufferIndex++`, `txBufferIndex++`, `i++`,.... This is probably because you've copied and pasted the code from the script, you wanted to remove the `+` from the code by doing a find/replace and you have also removed the `+` signs from the increments unintentionally. Fix that and check if it works already. Confirm if that's the case, if not, make a new commit with the index increments fixed and I'll take another look.

**Comment by Laiaborrell** [comment]
Thankou, might have forgotten them when doing again the session to fix the first error. I have placed again the increments. It seems like it is working now. 
Laia

**Comment by Albert-Alvarez** [comment]
Glad to hear that! I close the issue since it has been resolved 😉
